### PR TITLE
feat: improved the PR template

### PR DESCRIPTION
**Pull request by laxyapahuja**
## Description
Improved the existing Pull Request Template.

## Screenshots
Making a PR
image

Preview
image

image

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-Please Add Screenshots If any UI changes.
+## Screenshots
+<!--Please Add Screenshots or Screen Recordings which show the changes you made.-->
 
-Please Add Summary of what changes you have made.(Optional)
+## Description
+<!--Please Add Summary of what changes you have made.-->
 
-Please make sure these boxes are checked before submitting your pull request - thanks!
+##
```
Comment (luckyman20) [inline .github/PULL_REQUEST_TEMPLATE.md]: Why these are needed here?

Comment (laxyapahuja) [reply .github/PULL_REQUEST_TEMPLATE.md]: Keeping this creates a line which divides the area correctly.

Comment (luckyman20) [reply .github/PULL_REQUEST_TEMPLATE.md]: Can you please add the screenshot for that too?

Comment (laxyapahuja) [reply .github/PULL_REQUEST_TEMPLATE.md]: already uploaded. you can see a thin line which divides the PR template before the checkboxes are there.

**Pull request merged**